OracleDatabase/SingleInstance/extensions/prebuiltdb/README.md

```diff
@@ -0,0 +1,39 @@
+# Pre-built Database Extension
+
+This extension creates an additional layer having a pre-built database on top of the [base Oracle SingleInstance Database image](../../README.md).
```

Comment:
Isn't Single Instance two words?  Also the sentence needs clarification.

Reply:
Fixed.